src/lib.rs

```diff
@@ -132,12 +163,12 @@ macro_rules! dlopen_external_library(
         $(functions: $(fn $fname: ident($($farg: ty),*) -> $fret:ty),+,)|*
         $(varargs: $(fn $vname: ident($($vargs: ty),+) -> $vret: ty),+,)|*
     ) => (
-        dlopen_external_library!(__struct,
+        dlib::dlopen_external_library!(__struct,
```

Comment:
I think this should use the `$crate::` special token, which will correctly resolve even if dlib is renamed by the crate using it.